robohive/envs/env_base.py

```diff
@@ -702,7 +733,7 @@ def examine_policy(self,
             ep_rwd = 0.0
             while t < horizon and done is False:
                 a = policy.get_action(o)[0] if mode == 'exploration' else policy.get_action(o)[1]['evaluation']
-                next_o, rwd, done, env_info = self.step(a)
+                next_o, rwd, done, *_, env_info = self.step(a)
```

Comment:
does these *_ work?

Reply:
yes, it is a more flexible way to manage the fact that the step function might return >= 4 values